src/layout/Header/NavMenuDrop/index.jsx

```diff
@@ -0,0 +1,40 @@
+import { Col, Row } from "antd";
+import React from "react";
+import "./style.scss";
+
+const NavMenuDrop = ({ list, imgRight }) => {
+  return (
+    <div className="nav-menu-drop">
+      <Row className="item-drop-down">
+        {list.map((item) => {
+          return (
+            <Col lg={6} key={item.title}>
+              <div className="title">
+                <a className="item-link" href="#!">
+                  {item.title}
+                </a>
+              </div>
+              {item.items.map((value, index) => {
+                return (
+                  <div className="item" key={index}>
+                    <a className="item-link" href="#!">
+                      {value.title}
+                    </a>
+                  </div>
+                );
+              })}
+            </Col>
+          );
+        })}
```

Comment:
nếu viết gọn hơn thì có thể viết như này
```suggestion
        {list.map(({ title, items }) => (
            <Col lg={6} key={title}>
              <div className="title">
                <a className="item-link" href="#!">
                  {title}
                </a>
              </div>
              {items.map(({ title }) => (
                  <div className="item" key={title}>
                    <a className="item-link" href="#!">
                      {title}
                    </a>
                  </div>
                ))}
            </Col>
          ))}
```